x-pack/plugins/observability_solution/apm/public/components/routing/templates/apm_main_template/index.tsx

```diff
@@ -168,47 +163,49 @@ export function ApmMainTemplate({
     </EuiFlexGroup>
   );
 
-  const pageTemplate = showCustomEmptyState ? (
-    <CustomNoDataTemplate isPageDataLoaded={isLoading === false} noDataConfig={noDataConfig} />
-  ) : (
-    <ObservabilityPageTemplate
-      noDataConfig={
-        shouldBypassNoDataScreen ||
-        serviceInventoryViewLocalStorageSetting === ServiceInventoryView.entity
-          ? undefined
-          : noDataConfig
-      }
-      isPageDataLoaded={isLoading === false}
-      pageHeader={{
-        rightSideItems,
-        ...pageHeader,
-        pageTitle: pageHeaderTitle,
-        children: (
-          <EuiFlexGroup direction="column">
-            {isEntityCentricExperienceSettingEnabled &&
-            showEnablementCallout &&
-            selectedNavButton === 'allServices' ? (
-              <EntityEnablement
-                label={i18n.translate('xpack.apm.eemEnablement.tryItButton.', {
-                  defaultMessage: 'Try our new experience!',
-                })}
-                tooltip={i18n.translate('xpack.apm.entityEnablement.content', {
-                  defaultMessage:
-                    'Our new experience combines both APM-instrumented services with services detected from logs in a single service inventory.',
-                })}
-              />
-            ) : null}
-            {showServiceGroupsNav && selectedNavButton && (
-              <ServiceGroupsButtonGroup selectedNavButton={selectedNavButton} />
-            )}
-          </EuiFlexGroup>
-        ),
-      }}
-      {...pageTemplateProps}
-    >
-      {children}
-    </ObservabilityPageTemplate>
+  const [dismissedInventoryCallout, setDismissedInventoryCallout] = useLocalStorage(
+    `apm.dismissedInventoryCallout`,
+    false
   );
 
-  return <EnvironmentsContextProvider>{pageTemplate}</EnvironmentsContextProvider>;
+  const showInventoryCallout =
+    !dismissedInventoryCallout &&
+    isEntityCentricExperienceSettingEnabled &&
+    selectedNavButton === 'allServices';
```

Comment:
I'm not sure about this. Should we just show it for all tabs? 
cc: @kpatticha / @roshan-elastic

Reply:
@cauemarcondes - Do you mean all tabs in the service inventory (i.e. service map too)?

Or, do you mean service views?

Happy for it to be on the service inventory + the service map too (but not the service views themselves)

Reply:
We already show it on the **Services inventory** and **Service maps**. We only hide it on the Service Groups page. My suggestion is to show it on the **Service Groups** page too.

Reply:
I don't have a strong opinion on that. It doesn't hurt to show it also on the Service groups page. (as a bonus: we'll fix the layout shift 😄 ) 

https://github.com/user-attachments/assets/ff5d8464-6a5a-428f-8965-fcbd8c76388c

Reply:
Makes sense. If its makes things simpler for you - show it @cauemarcondes @kpatticha. From a user perspective, it doesn't really matter either way.